packages/lit-dev-content/site/docs/ssr/authoring.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Authoring components for Lit SSR
+eleventyNavigation:
+  key: Authoring components
+  parent: Server rendering
+  order: 4
+---
+
+{% labs-disclaimer %}
+
+Lit's approach to rendering web components in a server environment places some restrictions on component code to achieve efficient server rendering. When authoring components, keep in mind these considerations to ensure they are compatible with Lit SSR.
+
+Note: The restrictions listed on this page are subject to change as we make improvements to Lit SSR. If you would like to see a certain use case supported, please [file an issue](https://github.com/lit/lit/issues/new/choose) or start a [discussion](https://github.com/lit/lit/discussions) thread.
+
+## Considerations
```

Comment:
As a point of style, we try to avoid empty headings (that is, a heading followed immediately by another heading with no text in between.

In this case, I'm not sure this grouping adds anything, so I'd recommend cutting it.